Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -290,6 +295,125 @@ public override List<Cash> GetCashBalance()
 
             return list;
         }
+
+        /// <summary>
+        /// Gets the history for the requested security
+        /// </summary>
+        /// <param name="request">The historical data request</param>
+        /// <returns>An enumerable of bars covering the span specified in the request</returns>
+        public override IEnumerable<BaseData> GetHistory(HistoryRequest request)
+        {
+            // GDAX API only allows us to support history requests for TickType.Trade
+            if (request.TickType != TickType.Trade)
+            {
+                yield break;
+            }
+
+            if (request.EndTimeUtc < request.StartTimeUtc)
+            {
+                Log.Error("GDAXBrokerage.GetHistory(): The start date must precede the end date, no history returned");
```

Comment:
Instead of `Log.Error`, maybe using the brokerage message system would be more useful to users can see the errors.

Reply:
👍 Will update to call `OnMessage` with warning event